common/src/main/java/com/wynntils/wrappedscreens/trademarket/TradeMarketSearchResultScreen.java

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wrappedscreens.trademarket;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Models;
+import com.wynntils.handlers.wrappedscreen.WrappedScreen;
+import com.wynntils.handlers.wrappedscreen.type.WrappedScreenInfo;
+import com.wynntils.screens.base.TextboxScreen;
+import com.wynntils.screens.base.TooltipProvider;
+import com.wynntils.screens.base.WynntilsContainerScreen;
+import com.wynntils.screens.base.widgets.BasicTexturedButton;
+import com.wynntils.screens.base.widgets.ItemSearchHelperWidget;
+import com.wynntils.screens.base.widgets.ItemSearchWidget;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.services.itemfilter.type.ItemSearchQuery;
+import com.wynntils.utils.MathUtils;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.wrappedscreens.trademarket.widgets.PresetButton;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.components.events.GuiEventListener;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.item.ItemStack;
+
+public class TradeMarketSearchResultScreen extends WynntilsContainerScreen<ChestMenu> implements WrappedScreen {
+    // Constants
+    private static final int FAKE_CONTAINER_ID = 454545;
+    private static final CustomColor LABEL_COLOR = CustomColor.fromInt(0x404040);
+    private static final ResourceLocation CONTAINER_BACKGROUND =
+            new ResourceLocation("textures/gui/container/generic_54.png");
+    private static final int SCROLL_AREA_HEIGHT = 110;
+    private static final int ITEMS_PER_PAGE = 54;
+
+    // Info
+    private final TradeMarketSearchResultHolder parent;
+    private final WrappedScreenInfo wrappedScreenInfo;
+
+    // Widgets
+    private ItemSearchWidget itemSearchWidget;
+
+    // State
+    private Component currentState = Component.empty();
+
+    // Scrolling
+    private int scrollOffset = 0;
+    private boolean holdingScrollbar = false;
+
+    protected TradeMarketSearchResultScreen(WrappedScreenInfo wrappedScreenInfo, TradeMarketSearchResultHolder parent) {
```

Comment:
```suggestion
    protected TradeMarketSearchResultScreen(WrappedScreenInfo wrappedScreenInfo, TradeMarketSearchResultHolder holder) {
```

Maybe search/replace everywhere?